portal/src/pages/index.vue

```diff
@@ -27,15 +27,13 @@
           </div>
           <Search
             v-if="!$root.isDemoEnvironment()"
-            v-model="searchText"
```

Comment:
More changes that might be relevant for @naftalivanderloon, again multiple changes in the entire file